dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -497,42 +499,29 @@ public void removeAttachedFile(WorkflowTask task, String fileInode) throws DotDa
 		final boolean isNew  = !UtilMethods.isSet(contentlet.getInode());
 		boolean canLock      = false;
 		boolean isLocked     = false;
-		String lockedUserId  =  null;
 
 		try {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getStructure"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)